CMakeLists.txt

```diff
@@ -419,17 +428,19 @@ else()
     add_executable(${EXE} ${EXE_SRC})
   elseif(NOT MSVC_VERSION)
     add_executable(${EXE} ${EXE_SRC} llarp/win32/version.rc)
-  else() 
+  else()
     add_executable(${EXE} ${EXE_SRC})
   endif(NOT WIN32)
 
   add_log_tag(${EXE})
 
-  install(TARGETS ${EXE} RUNTIME DESTINATION bin)
-  if(WIN32)
-    install(PROGRAMS ${CMAKE_SOURCE_DIR}/lokinet-bootstrap.exe DESTINATION bin)
-  else()
-    install(PROGRAMS ${CMAKE_SOURCE_DIR}/lokinet-bootstrap DESTINATION bin)
+  if(NOT IOS)
```

Comment:
This is unnecessary and possibly mistaken?

The binaries installed here don't have to be included in an iOS extension (this can be controlled from Xcode, AppCode, or whatever you're using), and you may wish to run these binaries internally.